models/public/faster_rcnn_inception_v2_coco/faster_rcnn_inception_v2_coco.md

```diff
@@ -2,7 +2,7 @@
 
 ## Use Case and High-Level Description
 
-Faster R-CNN with Inception v2. Used for object detection. For details, see the [paper](https://arxiv.org/pdf/1801.04381.pdf).
+Faster R-CNN with Inception v2. Used for object detection. For details, see the [paper](https://arxiv.org/pdf/1506.01497v3.pdf).
```

Comment:
Shouldn't a similar change be made to `faster_rcnn_inception_resnet_v2_atrous_coco`, as well?

Reply:
updated, thanks